addons/l10n_fr/data/account_chart_template_data.xml

```diff
@@ -314,56 +314,56 @@
     </record>
 
     <record id="pcg_13911" model="account.account.template">
-      <field name="name">Etat</field>
+      <field name="name">Subventions d'investissement - État</field>
```

Comment:
Checking http://www.plancomptable.com/titre-IV/liste_des_comptes_sd.htm, it looks like both `1311` and `13911` are "Subventions d'investissement - Subventions d'équipement - Etat". How about "Subventions d'investissement inscrites au compte de résultat" ? Or is it too long for the reports ?